pages/draft/2020-12/migration-notes.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Draft 2019-09 to Draft 2020-12
+section: docs
+---
+
+### Introduction
+
+JSON Schema Draft 2020-12 introduces key improvements over Draft 2019-09, focusing on improved modularity and cross-referencing capabilities. This means improvements for schema resuse, being able to define features a schema supports, etc. This guide covers all major changes and how to migrate from Draft 2019-09 to Draft 2020-12.
+
+### Keyword changelog
+
+| Keyword (Draft 2019-09) | Keyword (Draft 2020-12) | Specification | Keyword Type | Behavior Details |
+| --- | --- | --- | --- | --- |
+| $anchor | remained $anchor | Core |  | In 2019-09, $anchor could use a colon and not start with an underscore, but in 2020-12, the upgrade in syntax rule can't use a colon. |
```

Comment:
In the description, you are not enclosing $anchor in back ticks, but in other cases you always do. Can you cross check to ensure consistency?

Reply:
Thank you, updated.